x-pack/test/alerting_api_integration/basic/tests/alerts/gold_noop_alert_type.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getTestAlertData } from '../../../common/lib';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function emailTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+
+  describe('create gold noop alert', () => {
+    it('should return 403 when creating an gold alert', async () => {
+      const { body: createdAction } = await supertest
+        .post(`/api/actions/action`)
+        .set('kbn-xsrf', 'foo')
+        .send({
+          name: 'MY action',
+          actionTypeId: 'test.noop',
+          config: {},
+          secrets: {},
+        })
+        .expect(200);
+
+      await supertest
+        .post(`/api/alerts/alert`)
+        .set('kbn-xsrf', 'foo')
+        .send(
+          getTestAlertData({
```

Comment:
This would need to overwrite the `alertTypeId` to the one you created (`test.gold.noop`). This will also give you the error message you're looking for below.

**nit:** I would also remove the actions from the scenario as the `test.*` actions are all gold+. Otherwise, it would need to be `.server-log` or something along those lines.

```suggestion
          getTestAlertData({
            alertTypeId: 'test.gold.noop'
```